doc/src/man/litani-run-build.scdoc

```diff
@@ -60,6 +61,9 @@ parallel.
 	Return _0_ only if all pipelines were successful. See *RETURN CODE*
 	below.
 
+*--no-draw-dependency-graph*
+	Does not attempt to generate dependency graph.
```

Comment:
* I think this should be `--no-pipeline-dep-graph`, because there are other kinds of dependency graphs that we could generate in the future.
* Use the imperative: "Do not". I would write "Do not render dependency graphs for each pipeline onto the HTML individual pipeline pages. Pipeline graphs will also not be rendered if Graphviz is not installed".